tests/framework/utils/k8s_helper_test.go

```diff
@@ -1,140 +0,0 @@
-/*
-Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
Was this related to cassandra, or you just found dead code?

Reply:
Dead code from replacing the kubeconfig loading code.